apps/server/src/modules/common-cartridge/service/common-cartridge-import.service.ts

```diff
@@ -0,0 +1,21 @@
+import { CoursesClientAdapter } from '@infra/courses-client';
+import { Injectable } from '@nestjs/common';
+import { CommonCartridgeFileParser } from '../import/common-cartridge-file-parser';
+import { DEFAULT_FILE_PARSER_OPTIONS } from '../import/common-cartridge-import.types';
+
+@Injectable()
+export class CommonCartridgeImportService {
+	constructor(private readonly coursesClient: CoursesClientAdapter) {}
+
+	public async importFile(file: Buffer): Promise<void> {
+		const parser = new CommonCartridgeFileParser(file, DEFAULT_FILE_PARSER_OPTIONS);
+
+		await this.createCourse(parser);
+	}
+
+	private async createCourse(parser: CommonCartridgeFileParser): Promise<void> {
+		const courseName = parser.getTitle() || 'Untitled Course';

```

Comment:
SonarQube: Prefer using nullish coalescing operator (`??`) instead of a logical or (`||`), as it is a safer operator
```suggestion
		const courseName = parser.getTitle() ?? 'Untitled Course';
```